demo/init-container-failure-in-cluster.demo

```diff
@@ -0,0 +1,39 @@
+# Demo: inject fault to initContainer in an existing cluster such as minikube or kind
+
+kubectl get nodes
+
+# Prepare manifest files
+make manifests
+sed -i -e 's/\/kwok:v0.5.0-94-g9b555d3-dirty/registry.k8s.io\/kwok\/kwok:v0.5.1/g' artifacts/kwok.yaml
+clear
+
+# Deploy and install kwok
+kubectl apply -f artifacts/kwok.yaml
```

Comment:
```suggestion
kubectl apply -f https://github.com/kubernetes-sigs/kwok/releases/download/v0.5.2/kwok.yaml
```

I am not sure if it is better to use the deploy file directly in the releases, WDYT?